tests/foreman/ui/test_contenthost.py

```diff
@@ -1473,36 +1471,34 @@ def test_content_access_after_stopped_foreman(
 
     :CaseComponent: Infrastructure
 
-    :Assignee: lpramuk
-
     :parametrized: yes
     """
-    sat = foreman_service_teardown
-    org = sat.api.Organization().create()
+    org = target_sat.api.Organization().create()
     # adding remote_execution_connect_by_ip=Yes at org level
-    sat.api.Parameter(
+    target_sat.api.Parameter(
         name='remote_execution_connect_by_ip',
         value='Yes',
         parameter_type='boolean',
         organization=org.id,
     ).create()
-    lce = sat.api.LifecycleEnvironment(organization=org).create()
-    with sat:  # ensure the context section only uses this satellite
-        repos_collection = RepositoryCollection(
-            distro=DISTRO_RHEL7,
-            repositories=[
-                RHELAnsibleEngineRepository(cdn=True),
-                SatelliteToolsRepository(),
-                YumRepository(url=settings.repos.yum_1.url),
-                YumRepository(url=settings.repos.yum_6.url),
-            ],
-        )
-        repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
-        repos_collection.setup_virtual_machine(rhel7_contenthost, target_sat)
+    lce = target_sat.api.LifecycleEnvironment(organization=org).create()
+    repos_collection = RepositoryCollection(
+        distro=DISTRO_RHEL7,
+        repositories=[
+            YumRepository(url=settings.repos.yum_1.url),
+        ],
+    )
+    repos_collection.setup_content(org.id, lce.id, upload_manifest=False)
+    repos_collection.setup_virtual_machine(
+        rhel7_contenthost, target_sat, install_katello_agent=False
+    )
```

Comment:
This will be refactored as https://github.com/SatelliteQE/robottelo/pull/9497/files#diff-9021f84900ea80f6128124119a665669cecf60c29ec23da7bbb49611833e739fR1825-R1834

But wait for https://github.com/SatelliteQE/robottelo/pull/9497 to get merge :)

Reply:
Thanks all